spyder/app/mainwindow.py

```diff
@@ -2375,40 +2376,75 @@ def report_issue(self, traceback=""):
         revision = ''
         if versions['revision']:
             revision = versions['revision']
+        reminder_message = (
+            "**PLEASE READ:** Before reporting here, *please* carefully "
+            "read our **Troubleshooting Guide**: <{0!s}> and search the issue "
+            "page for your error/problem, as most "
+            "posted bugs are duplicates or easy fixes.\n\n"
+            "Also, if you do post here, please describe step by step in "
+            "detail what you were doing when the error occurred. "
+            "Otherwise, your issue will likely be closed after 7 days. "
+            "Thanks.\n\n"
+            "**Note:** Delete this message before submitting."
+            ).format(__trouble_url__)
+        if traceback:
+            reminder_message = ("IMPORTANT: Read troubleshooting guide at "
+                                "{0!s} before posting!"
+                                ).format(__trouble_url_short__)
+        else:
+            traceback = "PASTE HERE"
         issue_template = """\
-## Description
+%s

```

Comment:
Let's use `format` here, instead of simple string replacement because now it's really hard to understand what each replacement corresponds to. So this should be (if I'm correct) `{reminder}`, instead of `%s`.

Reply:
Will do, thanks. I typically use ``format`` myself but didn't want to mess with the existing code without being instructed to or a strong reason.